src/components/AcceptAllRejectAllToggle.tsx

```diff
@@ -94,9 +99,13 @@ export function AcceptAllRejectAllToggle({
                 ? messages.doNotSellOptedIn
                 : messages.doNotSellOptedOut,
             )}
+            initialFocus
           />
-
-          <p className="paragraph">
+          <p
+            className="paragraph"
+            role={'status'}
```

Comment:
just a style nit, but this could probably be `role="status"`. I think the monorepo would automatically correct this.